### DISCUSSION TITLE
how do I remove freshrss feed updates messages from systemd/journal

### DISCUSSION THREAD

**Discussion by rezad1393** [FreshRSS/FreshRSS#6499] [General]
how do I remove freshrss feed updates messages from systemd/journal?

they kinda pollute the systemd journal.

like this:

```
May 26 16:01:31 odroid FreshRSS[10616]: FreshRSS Favicon GET https://sourceforge.net/
May 26 16:01:33 odroid FreshRSS[13798]: FreshRSS Favicon GET https://github.githubassets.com/favicons/favicon.svg
May 26 16:01:46 odroid FreshRSS[10616]: FreshRSS Favicon GET https://sourceforge.net/favicon.ico
May 26 16:02:28 odroid FreshRSS[10616]: SimplePie GET https://sourceforge.net/p/clover-mipad4/activity/feed
```

I already get a periodic cron jobs for feeds updates in journal msgs.

**Comment by Alkarex** [comment]
There is an environment variable `COPY_SYSLOG_TO_STDERR`, which is `On` by default in Docker, and which you can set to `Off`

https://github.com/FreshRSS/FreshRSS/blob/b7cc8142380ccf95d2ec886706e897dcfa9c946c/Docker/Dockerfile#L55

And there is likewise `COPY_LOG_TO_SYSLOG`

https://github.com/FreshRSS/FreshRSS/blob/b7cc8142380ccf95d2ec886706e897dcfa9c946c/Docker/Dockerfile#L54

**Reply by rezad1393** [reply]
how can I change that in normal install (not docker) ?

is there an option in config,php?

**Reply by Alkarex** [reply]
Yes, you can define them in a local `./constants.local.php` on the model of 

```php
<?php
define('COPY_LOG_TO_SYSLOG', 'Off');
```

But in non-Docker environment, they are supposed to be off by default already.

In you case, it might then be the `simplepie_syslog_enabled` variable in your `./data/config.php`, which should have your interest.

**Reply by rezad1393** [reply]
"./constants.local.php" is that a file I create in data folder?
because I see a file ./constants.php in root folder of app but not in data folder where the user config are.

**Reply by math-GH** [reply]
yes. This file needs to be created manually.

See the manual too (regarding the file `constants.local.php`) : https://freshrss.github.io/FreshRSS/en/admins/17_configs_not_ui.html#application-wide-constants

**Reply by rezad1393** [reply]
thank you.
but why is the config in root?
would it not be better for it to be in the data folder?
make cleanup easier.

**Reply by Alkarex** [reply]
`simplepie_syslog_enabled` is in `./data/config.php`.
`./constants.local.php` is used for higher level configuration, such as defining the location of the `data` folder.
